### PR TITLE
Merge release/19.8 after code freeze

### PR DESCRIPTION
**Pull request by AliSoftware**
- [X] New empty entry created in `RELEASE-NOTES.txt` for next iteration.
- [X] `{jetpack_}metadata/release_notes.txt` extracted for WordPress and Jetpack.
- [X] `WordPress/jetpack_metadata/release_notes.txt` audited to remove any item not applicable for Jetpack.
- [X] Internal dependencies (FluxC, Login. Stories. About…) bumped to a new stable version in `build.gradle` if necessary.
   - ℹ️  Had to also include #16455 to fix a unit test failure on the FluxC commit I created a tag for
- [X] `WordPress/src/main/res/values/strings.xml` updated with strings from binary dependencies.
   - [X] 🆕  Now includes [the new `a8c-lib-src` attribute on all merged strings](https://github.com/wordpress-mobile/WordPress-Android/pull/16280)
- [X] New strings frozen/copied into `fastlane/resources/values/strings.xml`.
- [X] Version bumped in `version.properties`

### CODE REVIEW

File: WordPress/jetpack_metadata/release_notes.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
+* [*] Editor: Fix issue that caused previews on atomic sites to sometimes be out-of-date. [https://github.com/wordpress-mobile/WordPress-Android/pull/16332]
+* [*] Fixes Media Picker video selection Accessibility issue [https://github.com/wordpress-mobile/WordPress-Android/pull/16426]
+* [*] Block Editor: Latest Posts block: Add featured image settings [https://github.com/WordPress/gutenberg/pull/39257]
+* [*] Block Editor: Prevent incorrect notices displaying when switching between HTML-Visual mode quickly [https://github.com/WordPress/gutenberg/pull/40415]
+* [*] Block Editor: Embed block: Fix inline preview cut-off when editing URL [https://github.com/WordPress/gutenberg/pull/35326]
+* [*] Fixes signup magic link URL [https://github.com/wordpress-mobile/WordPress-Android/pull/16449]
```
Comment (AliSoftware) [inline WordPress/jetpack_metadata/release_notes.txt:8]: This one is specific to Jetpack — I kept it (but deleted the "Jetpack App" prefix [from the original line](https://github.com/wordpress-mobile/WordPress-Android/blob/0f0f347d306a30de8191dad720bb12e802bab8e6/RELEASE-NOTES.txt#L19)) in Jetpack Release Notes, but removed it from WordPress Release Notes. \cc @leandroalonso

File: WordPress/metadata/release_notes.txt [hunk 2]
```diff
@@ -1,5 +1,10 @@
-- Added Site Menu and Home tabs to My Site screen
-- Made Add Block button more visible when block editor opens
-- Removed media/audio block upload error text
-- Added Site Intent screen to site creation process
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
+* [*] On self-hosted sites, allow previewing unsaved changes to draft posts [https://github.com/wordpress-mobile/WordPress-Android/pull/16296]
```
Comment (AliSoftware) [inline WordPress/metadata/release_notes.txt:3]: Present in those WordPress Release Notes, but removed the line from the Jetpack ones — as it's about self-hosted sites which is not applicable to the Jetpack app. \cc @leandroalonso

File: WordPress/metadata/release_notes.txt [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+* [*] Editor: Fix issue that caused previews on atomic sites to sometimes be out-of-date. [https://github.com/wordpress-mobile/WordPress-Android/pull/16332]
+* [*] Fixes Media Picker video selection Accessibility issue [https://github.com/wordpress-mobile/WordPress-Android/pull/16426]
+* [*] Fixes self-hosted upload failure when the filename is non-latin [https://github.com/wordpress-mobile/WordPress-Android/pull/16421]
+* [*] Fixes self-hosted blurred featured image on post settings [https://github.com/wordpress-mobile/WordPress-Android/pull/16439]
```
Comment (AliSoftware) [inline WordPress/metadata/release_notes.txt:7]: Present in those WordPress Release Notes, but removed the 2 lines from the Jetpack ones — as it's about self-hosted sites which is not applicable to the Jetpack app. \cc @leandroalonso

File: WordPress/metadata/release_notes.txt [hunk 4]
```diff
@@ -1,5 +1,10 @@
-- Added Site Menu and Home tabs to My Site screen
-- Made Add Block button more visible when block editor opens
-- Removed media/audio block upload error text
-- Added Site Intent screen to site creation process
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
```
Comment (AliSoftware) [inline WordPress/metadata/release_notes.txt:2]: Kept this one in the WordPress release notes, as this is about _Jetpack-the-service_, not _Jetpack-the-app_, so I think it's still applicable to the WordPress app too. \cc @leandroalonso

**Review by AliSoftware — COMMENTED** [review]
@leandroalonso I think I audited the WordPress & Jetpack release notes correctly, but would love a confidence check nonetheless 🙂

**Comment by AliSoftware** [comment]
> Found 1 violations:
> 
> ### The PR caused the following dependency changes:
> ```diff
> -+--- org.wordpress:fluxc:{strictly trunk-889b815f5bfeddfc877f1bc6820fa72129b422c7} -> trunk-889b815f5bfeddfc877f1bc6820fa72129b422c7
> …
> ++--- org.wordpress:fluxc:{strictly 1.41.0} -> 1.41.0
> ```
> 
> _Please review and act accordingly_

This actually required us to fix a Unit Test via https://github.com/wordpress-mobile/WordPress-Android/pull/16455 that started failing after I pointed to the tag. This is my bad for making the `1.41.0` point to FluxC's `trunk` head without checking, instead of making it point to `889b815f5bfeddfc877f1bc6820fa72129b422c7` — the commit it was pointing to before the code freeze — leading to including a small FluxC API change (not yet used in any visible feature in the app, as that part is currently feature-flagged, but called in the Unit Tests) that made the test fail.

Kudos @irfano for spotting and fixing this quickly while I was doing the code freeze.

**Comment by leandroalonso** [comment]
@AliSoftware it all looks good to me, thanks for double-checking it!

**Review by oguzkocer — APPROVED** [review]

**Pull request merged**